tests/test_api.py

```diff
@@ -44,17 +42,31 @@ def test_create_manifestation(client, user):
         'copyright': {
             '@context': '<coalaip placeholder>',
             '@type': 'Copyright',
-            '@id': '',
         },
     }
     resp = client.post(url_for('manifestation_views.manifestationapi'),
                        data=json.dumps(payload),
                        headers={'Content-Type': 'application/json'})
     resp_dict = resp.json
-    assert bool(resp_dict['copyright']['rightsOf']) is True
-    assert bool(resp_dict['manifestation']['manifestationOfWork']) is True
+    copyright_ = resp_dict['copyright']
+    manifestation = resp_dict['manifestation']
+    work = resp_dict['work']
+
+    assert bool(copyright_['rightsOf']) is True
+    assert bool(manifestation['manifestationOfWork']) is True
+
+    # Check @ids
+    assert copyright_['@id'].startswith('../right/')
+    assert bool(copyright_['@id'].strip('../right/')) is True
+    assert bool(manifestation['@id']) is True
```

Comment:
If you wanted to be hard core you could import BDBC, serialize the tx and check it's id.